pkg/orchestrator/pod_test.go

```diff
@@ -522,6 +560,62 @@ func getScrubCases() map[string]struct {
 				Env: []v1.EnvVar{{Name: "password", Value: "********"}},
 			},
 		},
+		"command with sensitive arg": {
+			input: v1.Container{
+				Command: []string{"mysql"},
+				Args:    []string{"--password", "afztyerbzio1234"},
+			},
+			expected: v1.Container{
+				Command: []string{"mysql"},
+				Args:    []string{"--password", "********"},
+			},
+		},
+		"command with no sensitive arg": {
+			input: v1.Container{
+				Command: []string{"mysql"},
+				Args:    []string{"--debug", "afztyerbzio1234"},
+			},
+			expected: v1.Container{
+				Command: []string{"mysql"},
+				Args:    []string{"--debug", "afztyerbzio1234"},
+			},
+		},
+		"sensitive command with no sensitive arg": {
```

Comment:
Could we add one test cases where both args and command only have parameters without spaces in them?

Reply:
added them, 💯